hashets/hash_fs.go

```diff
@@ -0,0 +1,37 @@
+package hashets
+
+import (
+	"io/fs"
+)
+
+type hashFS struct {
+	fsys fs.FS
+	m    map[string]string
```

Comment:
Semantically, this should probably be a `Map`.

```suggestion
	m    Map
```